addon/components/utils/address-form.ts

```diff
@@ -70,10 +108,14 @@ export default class extends Component<UtilsAddressFormArgs> {
   }
 
   private _checkAddressValidity(): boolean {
```

Comment:
when we use a `private` keyword, we don't use `_`  as a method prefix. (that was one of the way of showing `private` methods in JS).
Can you please remove it ? 😇